QuizWiz.ApiService/Program.cs

```diff
@@ -1,21 +1,63 @@
+using Microsoft.AspNetCore.Identity;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.DependencyInjection;
+using QuizWiz.ApiService.Endpoints;
+using QuizWiz.ApiService.Services;
+using QuizWiz.Data.Context;
+using QuizWiz.Data.Model;
+using QuizWix.Application.Auth;
+using System;
+using System.Reflection;
+using System.Security.Claims;
+
 var builder = WebApplication.CreateBuilder(args);
 
 // Add service defaults & Aspire components.
 builder.AddServiceDefaults();
 
 // Add services to the container.
 builder.Services.AddProblemDetails();
+builder.Services.AddAuthorization();
+
+// Add EfCore
+builder.Services.AddDbContext<QuizWixContext>(options =>
```

Comment:
I have added a DI extension so we can move these registrations there. That would be cleaner I think.